### PR TITLE
solved 5.js

### PR DESCRIPTION
**Pull request by kinuesantos**

### CODE REVIEW

**Review by seemcat — APPROVED** [review]
I would stick with the standard 'solution/test' names just for consistency.

**Review by seemcat — APPROVED** [review]
Next time just stick with the standard names: solution/test for consistency throughout your repository.

File: solutions/5.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+const sayHi = (func,num, i=0)=>{
+  if(i==num){
```
Comment (seemcat) [inline solutions/5.js]: if(i<num){
func();
sayHi(func, num, i+1);
} 
***This way, there's no need to return anything.

Comment (seemcat) [reply solutions/5.js]: makes the code look cleaner...song made this comment on minesss

**Review by seemcat — CHANGES_REQUESTED** [review]

File: solutions/5.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+const sayHi = (func,num, i=0)=>{
+  if(i<num){
+    func();
+    return sayHi(func,num,i+1);
```
Comment (seemcat) [inline solutions/5.js:4]: you don't need to return. But this is optional change you don't need to

**Review by seemcat — APPROVED** [review]

**Review by evo777 — APPROVED** [review]

**Pull request merged**